docs/diracdoctools/__init__.py

```diff
@@ -8,17 +8,4 @@
 from __future__ import print_function
 
 # list of packages that should be mocked in sphinx
-# TODO: check if we can install more of these
-DIRAC_DOC_MOCK_LIST = ['_arc',
-                       'arc',
-                       'cmreslogging',
-                       'fts3',
-                       'gfal2',
-                       'git',
-                       'irods',
-                       'lcg_util',
-                       'MySQLdb',
-                       'pylab',
-                       'pythonjsonlogger',
-                       'stomp',
-                       ]
+DIRAC_DOC_MOCK_LIST = []
```

Comment:
Maybe keep the list available for people inheriting this list and not building on conda

Reply:
Do we want to support that? This mocking has always felt like a hack to workaround the fact that it was hard to install the dependencies.

Reply:
Do we want to stop supporting this, is the question.
Also this https://github.com/DIRACGrid/DIRAC/blob/integration/docs/source/DeveloperGuide/CodeDocumenting/index.rst#building-the-documentation needs an update